CHANGELOG.md

```diff
@@ -1,10 +1,24 @@
 ## 2.3.0
 
+### Breaking changes
+
+* `RealmResults.distinct()` returns a new `RealmResults` object instead of filtering on the original object (#2947).
```

Comment:
nope... it is very difficult to make it have both the old wrong behavior and the new async `Results` from object store.